psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -65,6 +68,10 @@
 #include <regex>
 #include <tuple>
 #include <functional>
+#include <memory>
+#include <map>
+#include <vector>
+#include <omp.h>
```

Comment:
This include needs to be wrapped with `#ifdef _OPENMP` and `#endif` otherwise it won't build on a standard Mac environment.

Actually, if no specific omp functions are being used (I didn't find any during a quick scan) then the include should be removed.

Reply:
Thank you for letting me know. I removed the `#include <omp.h>`